src/System.Private.CoreLib/shared/Internal/Win32/RegistryKey.cs

```diff
@@ -214,12 +215,12 @@ public unsafe string[] GetValueNames()
             return names.ToArray();
         }
 
-        public object GetValue(string name)
+        public object? GetValue(string name)
         {
             return GetValue(name, null);
         }
 
-        public object GetValue(string name, object defaultValue)
+        public object? GetValue(string name, object? defaultValue)
         {
             object data = defaultValue;
```

Comment:
I'm surprised this compiles. I'd have expected a warning when assigning the object? to object.

Reply:
You were correctly surprised 😄 the build is failing.